### ISSUE TITLE
--git switch not working on Debian-based systems

### ISSUE THREAD

**Issue by eggbean** [ogham/exa#761]
$ exa --version
exa v0.9.0

$ exa -l --git

On this particular computer, exa is not showing git status information in directories which are being tracked using git. The output is the same as it is without the --git switch.

This is on a particular computer running Pop!_OS 20.10 / bash 5.0.17 (essentially Ubuntu), while other computers are fine, using the same dotfiles.

Is there some reason why the --git switch does not work on this Pop!_OS laptop?

**Comment by ariasuni** [comment]
Do you have a `Git` entry if use the option `--header` like this: `exa -l --git --header`? Do you use exa at the root of your git repository? Did you try on other git repositories?

**Comment by eggbean** [comment]
The header switch doesn't change anything apart from add the header - still no git status information. I tried in the git root and further down into subdirectories. I have `--git` added to my alias and I have never seen the exa git functionality work on this Pop!_OS laptop. It's still a mystery why this is happening.

**Comment by ariasuni** [comment]
Ubuntu’s package has disabled git support, see [here](https://changelogs.ubuntu.com/changelogs/pool/universe/r/rust-exa/rust-exa_0.9.0-4/changelog). Since version 0.9.0-1 of the package:

> Disable git support because a test fail in libgit2-sys
>    I will fix it with the new version of libgit2-sys

According to [packages.ubuntu.com](https://packages.ubuntu.com/groovy/exa):

> Please consider [filing a bug](https://bugs.launchpad.net/ubuntu/+source/rust-exa/+filebug) or [asking a question](https://answers.launchpad.net/ubuntu/+source/rust-exa/+addquestion) via Launchpad before contacting the maintainer directly.

I think that exa should warn the user when the feature is disabled, so I’ll have a look at that. In the meantime, I guess someone should try to ask about the git support in the official Ubuntu package.

**Comment by eggbean** [comment]
Thanks. At least I know why it isn't working. I asked when it's likely to be re-enabled.

**Comment by ariasuni** [comment]
@eggbean did you have an answer? Maybe we need to update exa’s dependencies and publish a new version to fix this…

**Comment by eggbean** [comment]
@ariasuni Apparently it has been disabled on Debian testing, so there are going to be a lot of people who won't be able to use this very useful feature.

**Comment by verfriemelt-dot-org** [comment]
hm, thats a bit odd, on my debian sid system i can build it without any errors and all the tests are passing. so yeah

**Comment by eggbean** [comment]
I have noticed that with v0.9.0 an error occurs whenever with `--git` switch is used in a directory that has broken symlinks, similar to https://github.com/ogham/exa/issues/796. I think this might be the reason why the git feature was disabled by the Debian maintainer(s). 

I have compiled the current state of this repository and the error no longer happens, so wouldn't it be a good idea to make a new release so that the git feature works on on Debian-based distros?
